salt/annotations/config/etc-nginx-sites-enabled-annotations.conf

```diff
@@ -11,11 +11,15 @@ server {
     location ~ ^/app_{{ pillar.elife.env }}\.php(/|$) {
         fastcgi_split_path_info ^(.+\.php)(/.+)$;
         include fastcgi_params;
+        fastcgi_intercept_errors on;
         fastcgi_param SCRIPT_FILENAME $request_filename;
+        {% if pillar.annotations.containerized %}
+        fastcgi_pass localhost:9000;
```

Comment:
The container runs on port 9000 and this is forwarded from the host. The port is however closed to the outside world.